test/fixtures/standup/standup.js

```diff
@@ -0,0 +1,15 @@
+module.exports = () => {
+  return [
+    {
+      timestamp: "2023-04-20T16:23:58Z",
+      yesterday: "Did something",
+      today: "will do something",
+      blockers: "nothing",
+    },
+    {
+      yesterday: "Did something",
+      today: "will do something",
+      blockers: "nothing",
```

Comment:
we are missing `timestamp` here

Reply:
Yes, i deliberately did that to check invalid body